homeassistant/components/motionblinds_ble/const.py

```diff
@@ -0,0 +1,63 @@
+"""Constants for the MotionBlinds BLE integration."""
+
+from enum import StrEnum
+
+ATTR_CALIBRATION = "calibration"
+ATTR_CONNECTION_TYPE = "connection_type"
+ATTR_SPEED = "speed"
+ATTR_BATTERY = "battery"
+ATTR_CONNECT = "connect"
+ATTR_DISCONNECT = "disconnect"
+ATTR_FAVORITE = "favorite"
+ATTR_SIGNAL_STRENGTH = "signal_strength"
+
+CONF_LOCAL_NAME = "local_name"
+CONF_ADDRESS = "address"
```

Comment:
Remove this and use `homeassistant.const.CONF_ADDRESS` instead.

Reply:
Solved thanks